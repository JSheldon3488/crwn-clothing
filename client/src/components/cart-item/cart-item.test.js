
import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

it('Cart-Item Component test render and SnapShot', () => {
    const mockItem = {
        imageUrl: "https://i.ibb.co/0s3pdnc/adidas-nmd.png",
        price: 220,
        name: "Adidas NMD",
        quantity: 1 
    }
    
    expect(shallow(<CartItem item={mockItem}/>)).toMatchSnapshot();
})