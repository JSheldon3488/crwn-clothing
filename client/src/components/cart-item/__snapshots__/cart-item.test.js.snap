// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart-Item Component test SnapShot 1`] = `ShallowWrapper {}`;
